python/lsst/afw/detection/utils.py

```diff
@@ -23,7 +23,22 @@
 
 
 def writeFootprintAsDefects(fd, foot):
```

Comment:
This function is not included in the docs at the moment, because [`lsst.afw.detection` does not include `lsst.afw.detection.utils`](https://github.com/lsst/afw/blob/master/python/lsst/afw/detection/__init__.py). If you think it's worth it, you could add an `.. automodapi:: lsst.afw.detection.utils` entry for it; see `afw.table` for an example.

Reply:
For this initial round of review I left it out because I had tried adding that before, but the build broke because it complains about a missing doctree file.

Full details:

> FileNotFoundError: [Errno 2] No such file or directory: '/project/kherner/afw/doc/_build/doctree/py-api/lsst.afw.detection.utils.footprintToBBoxList.doctree'

What's interesting is that lsst.afw.detection.footprintToBBoxList.doctree does get created, but not a version with the .utils added. The relevant line in the code (python/lsst/afw/detection/utils.py) seems to be 

`from . import footprintToBBoxList`

So I suspect I need to tweak that somehow to get this all to play nicely together.

Reply:
Figured out a workaround: adding `:skip: footprintToBBoxList` allows everything to build and gets the missing function included.

Reply:
Ah, that's because the file doesn't have an `__all__` definition. I suggest adding it even if you have a workaround, since it's good practice (though it doesn't seem to be in the style guide 😕).

Reply:
Good catch, thanks! Adding `__all__ ["writeFootprintAsDefects"]` fixed it and the workaround is no longer necessary.